CONTRIBUTING.md

```diff
@@ -1,5 +1,24 @@
 # Contributing
 
+We'd love for you to contribute to our source code and to make **Webex Javascript SDK** even better than it is today!
+If you would like to contribute to this repository by adding features, enhancements or bug fixes, you must follow our process:
+
+  1. [Create an issue](https://github.com/webex/webex-js-sdk/issues) to propose your solution _before_ you get coding
+  2. Let core members know about your proposal by posting a message in the [contributor's Webex Teams space](https://eurl.io/#Bk9WGfRcB)
+  3. A core member will review your proposal and if necessary may suggest to have a meeting to better understand your approach
+      - You are welcomed you join our [weekly review meeting](https://cisco.webex.com/m/f4ebbec6-c306-49ca-83f4-fb2d098fc946) (Thursdays, 11:30a-12:30p PST) to propose your contribution as well
+  4. If your proposal is approved you should start coding at this point
+  5. We recommend opening a draft PR to receive feedback before finalizing your solution
+      - When opening a draft PR, specify with PR comments where in the code you would like to get feedback
+  6. Before opening a PR ensure **all** [PR guidelines](#submitting-a-pull-request) are followed
+  7. Let core members know about your PR by posting a message in the [contributor's Webex Teams space](https://eurl.io/#Bk9WGfRcB)
+  8. The PR will be reviewed by core members and leave feedback, if any
```

Comment:
This statement changes the subject half way through the sentence. It reads as though `The PR` `will` `leave feedback, if any`. Also mixed tense `future-past` to `future-simple`.
```
[subj] [verb----------]    [object----]     [ver] [object]  [cond]
The PR will be reviewed by core members and leave feedback, if any
```
try:
```
Future simple tense with compound predicates
[subject---] [mt] [predicate------------] [&] [predicate--------------------]
[subject---] [tv] [verb] [object--------] [&] [verb-] [object] [conditional-]
Core members will review the pull request and provide feedback when necessary
```